### ISSUE TITLE
Support wasm

### ISSUE THREAD

**Issue by sah4ez** [sah4ez/now#51]
In [the branch](https://github.com/sah4ez/pspk/tree/feature/webasm) begin work for migration to wasm for encryption/decryption. But I found a problem with support of goroutines in was, they didn't work with subcommand.
So, I think we can generate the separate modules in wasm and use this code in javascript applications.

Refactoring plan:
- [x] publish to the server
- [x] ephemeral encryption
- [x] ephemeral decryption
- [x] loading all keys
- [x] generation of QR code
- [ ] download QR code
- [ ] encryption with qrcode pub key
- [ ] decryption with qrcode priv key

**Comment by sah4ez** [comment]
@marti1125 If you have expertise in wasm, it would be great if you look at this issue.

**Comment by marti1125** [comment]
:exclamation:  no much experience in https://webassembly.org/ but I played with some examples. If you have any examples ?

**Comment by sah4ez** [comment]
[There](https://github.com/sah4ez/pspk/blob/feature/webasm/Makefile#L30), I think to add all subcommands of [this](https://github.com/sah4ez/pspk/tree/feature/webasm/cmd/cli) and write a [wrappers](https://github.com/sah4ez/pspk/blob/feature/webasm/cmd/wasm/publish/main.go)

**Comment by sah4ez** [comment]
But, I don't know how to solve the problem with imports and using many modules in javascript.

**Comment by marti1125** [comment]
here you are a implementation https://github.com/golang/go/wiki/WebAssembly.
what is the final result? or what do you spec using wasm?

**Comment by sah4ez** [comment]
Yes, I using this implementation. The goal, use wasm pspk in JavaScript, instead of vanilla js crypto. And use the one code base.

**Comment by sah4ez** [comment]
@marti1125 Good news!
I wrote the example `publish` [method](https://github.com/sah4ez/pspk/blob/feature/webasm/cmd/wasm/publish/main.go#L15) that successfully worked with the server. 
You can see [there](https://github.com/sah4ez/pspk/blob/feature/webasm/web_wasm/index.html#L87) call the method from wasm.

So, we need to rewrite all methods CLI for work with wasm and DOM HTML.

**Comment by marti1125** [comment]
awesome!!

**Comment by sah4ez** [comment]
@marti1125 I merged the master branch to the feature/webasm. And did successfully test for js code and with wasm.

**Comment by sah4ez** [comment]
Also, I added plan of refactoring for js part of the project.

**Comment by sah4ez** [comment]
Did [encrypt](https://github.com/sah4ez/pspk/blob/feature/webasm/cmd/wasm/eencrypt/main.go)/[decrypt](https://github.com/sah4ez/pspk/blob/feature/webasm/cmd/wasm/edecrypt/main.go) function for browser. And started work on loading [keys](https://github.com/sah4ez/pspk/blob/feature/webasm/cmd/wasm/keys/main.go) by regex. Had an error with renders keys in the table, maybe  to render need use golang and pure html.
